docs/tempo/website/grafana-agent/tail-based-sampling.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Tail-based sampling"
+---
+
+# Tail-based sampling
+
+Tempo aims to provide an inexpensive solution that makes 100% sampling possible.
+However, sometimes constraints will make a lower sampling percentage necessary or desirable,
+such as runtime or egress traffic related costs.
+Probabilistic sampling strategies are easy to implement,
+but also run the risk of discarding relevant data that you'll later want.
+
+In tail-based sampling, sampling decisions are made at the end of the workflow.
+The Grafana Agent groups spans by trace ID and makes a sampling decision based on the data contained in the trace.
+For instance, inspecting if a trace contains an error.
+
+To group spans by trace ID, the Agent buffers spans for a given time,
+after which it will consider the trace complete.
+Longer running traces will be split into more than one.
+Waiting longer times will increase the memory overhead of buffering.
```

Comment:
```suggestion
To group spans by trace ID, the Agent buffers spans for a configurable amount of time,
after which it will consider the trace complete.
Longer running traces will be split into more than one.
However, waiting longer times will increase the memory overhead of buffering.
```